registry/storage/linkedblobstore.go

```diff
@@ -31,6 +32,13 @@ type linkedBlobStore struct {
 var _ distribution.BlobStore = &linkedBlobStore{}
 
 func (lbs *linkedBlobStore) Stat(ctx context.Context, dgst digest.Digest) (distribution.Descriptor, error) {
+	tombstoneExists, err := lbs.tomb.tombstoneExists(ctx, lbs.repository.Name(), dgst)
```

Comment:
This check is here to prevent a fallthough to a cachedBlobStatter which does not have knowledge of tombstones and will therefore not return correctly for Stat